io.openems.backend.alerting/src/io/openems/backend/alerting/scheduler/MessageScheduler.java

```diff
@@ -26,14 +37,13 @@ public MessageScheduler(Handler<T> handler) {
 	 * @param msg to add
 	 */
 	public void schedule(T msg) {
-		if (msg == null) {
-			return;
-		}
-		synchronized (this) {
-			this.messageForId.computeIfAbsent(msg.getId(), (key) -> {
-				this.queue.add(msg);
-				return msg;
-			});
+		if (msg != null) {
```

Comment:
I consider the previous way best practice, as it reduces required indentation. Please revert.